bip-0009/assignments.mediawiki

```diff
@@ -0,0 +1,37 @@
+==Deployments==
+
+List of proposed deployments.
+
+State can be defined, active, failed. Dates are in UTC.
+
+{| class="wikitable"
```

Comment:
That's slightly complicated since bits will become free earlier than expiry once activated. Maybe we should keep it simple... or simply ~~strikeout~~ the bit once activated?